zerver/tests/test_reactions.py

```diff
@@ -84,6 +90,12 @@ def test_zulip_emoji(self) -> None:
         self.assert_json_success(result)
         self.assertEqual(200, result.status_code)
 
+        reaction_info.pop('reaction_type')
+        result = self.api_post(sender, '/api/v1/messages/2/reactions',
+                               reaction_info)
+        self.assert_json_success(result)
+        self.assertEqual(200, result.status_code)
```

Comment:
Can you add a commit extending these tests to actually verify the right `Reaction` rows got created, beyond just checking for a 200 status code?  These tests are not actually vierfying that it added a reaction; only that the API request returned status 200 :(.